bc_obps/compliance/service/supplementary_version_service.py

```diff
@@ -71,16 +71,32 @@ def handle(
 #         # Handle decreased obligation logic
 #         pass
 
-# # Concrete strategy for no significant change
-# class NoChangeHandler:
-#     def can_handle(self, new_summary: ReportComplianceSummary, previous_summary: ReportComplianceSummary) -> bool:
-#         # Return True if no significant change in excess emissions
-#         return False # return False until implemented
+# Concrete strategy for no significant change
+class NoChangeHandler:
+    def can_handle(self, new_summary: ReportComplianceSummary, previous_summary: ReportComplianceSummary) -> bool:
+        if (
+            new_summary.excess_emissions == previous_summary.excess_emissions
+            and new_summary.credited_emissions == previous_summary.credited_emissions
```

Comment:
The ticket talks about `earned_credits` but I ~think `credited_emissions` also works for this check?